### DISCUSSION TITLE
Release 0.9

### DISCUSSION THREAD

**Discussion by dufoli** [quarkiverse/quarkus-cxf#256] [General]
Hello,

I am thinking to release because we have quarkus 2.0 and Uber jar / native issue fixed. Wdyt?

We can optimize generation of clients if it is only used. This is the only issue I found useful to integrate.
The discution on health check lead to no real needs except disable Wsdl with config. We can do it too.

I do not see any other important topic.

Maybe add customization snippets in doc or add configuration to avoid code but not sure...

**Comment by dufoli** [comment]
@shumonsharif @famod

**Comment by famod** [comment]
Yeah, I think we should give that health topic more time to "bake".

I'll try to have a closer look at the client generation optimization.
Apart from this, that InjectMock topic would be interesting to my team (@mickroll is a colleague of mine).

**Comment by shumonsharif** [comment]
I would prefer we release 0.9 with the Quarkus 2.0 and other smaller fixes sooner than later.  With a major Quarkus upgrade, there could possibly be other issues lurking beneath the surface ....  :)  The more time we have folk testing this, the better off we are.

We can always release 0.9.x versions whenever we have the client generation optimization, health check and other issues / features addressed.

So @dufoli Please release away!!

**Reply by famod** [reply]
Alright, I fine with that, hoping for a "timely" 0.9.1 or 0.10.0. 🙂

**Comment by shumonsharif** [comment]
Release 0.9 is done ... may still need a bit of time to show up in [maven central](https://search.maven.org/artifact/io.quarkiverse.cxf/quarkus-cxf) (not sure how long it typically takes) ...

Also, we seem to have a ton of (possibly unused) branches ... @dufoli can we clean some of these up?
https://github.com/quarkiverse/quarkus-cxf/branches

**Reply by dufoli** [reply]
OK for me

**Comment by tpodg** [comment]
When should we expect next release? There were already some fixes, or are we waiting for some specific feature / fix?
